include/v4l2_camera/rate_bound_status.hpp

```diff
@@ -175,17 +157,16 @@ class RateBoundStatus : public diagnostic_updater::DiagnosticTask
     std::unique_lock<std::mutex> lock(lock_);
 
     // classify the current observation
-    StateHolder frame_result;
+    DiagnosticStatus_t frame_result;
```

Comment:
`Variable 'frame_result' is not initialized (fix available)clang-tidy[cppcoreguidelines-init-variables](https://clang.llvm.org/extra/clang-tidy/checks/cppcoreguidelines/init-variables.html)`
```suggestion
    DiagnosticStatus_t frame_result{};
```